### PR TITLE
OC-293: Upgrades Flyway version.

### PR DESCRIPTION
**Pull request by robindenadel**

### CODE REVIEW

**Comment by robindenadel** [comment]
The failing tests are going to be fixed within the FLEX-2888 story

File: osgp-platform-test/pom.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@
 
     <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-platform-test/pom.xml]: Remove unused dependency?

File: parent-integration-tests/pom.xml [hunk 2]
```diff
@@ -497,7 +497,7 @@
 
       <!-- Flyway (DB Migration) -->
       <dependency>
-        <groupId>com.googlecode.flyway</groupId>
+        <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline parent-integration-tests/pom.xml]: Remove unused dependency?

File: osgp-platform-test/pom.xml (hunk 1 above)
Comment (robindenadel) [inline osgp-platform-test/pom.xml]: ok

File: parent-integration-tests/pom.xml (hunk 2 above)
Comment (robindenadel) [inline parent-integration-tests/pom.xml]: ok

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/Integration-Tests?pullRequest=186) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: robindenadel<br/>:x: web-flow

**Comment by smvdheijden** [comment]
retest this please

**Comment by robindenadel** [comment]
retest this please

**Comment by smvdheijden** [comment]
retest this please

**Pull request merged**